src/cart/utils/memcheck-cart.supp

```diff
@@ -81,6 +81,82 @@
    fun:crt_hg_ctx_init
    ...
 }
+{
+   FI leak 6
+   Memcheck:Leak
+   match-leak-kinds: possible
+   ...
+   obj:/usr/lib64/librdmacm.so.1.1.22.1
+   ...
+}
+{
+   FI leak 7
+   Memcheck:Leak
+   match-leak-kinds: indirect
+   ...
+   fun:fi_ini
+   fun:fi_getinfo@@FABRIC_1.3
+   ...
+}
+{
+   FI leak 8
+   Memcheck:Leak
+   match-leak-kinds: possible
+   ...
+   fun:fi_ini
+   fun:fi_getinfo@@FABRIC_1.3
+   ...
+}
+{
+   FI leak 9
+   Memcheck:Leak
+   match-leak-kinds: reachable
+   ...
+   fun:fi_ini
+   fun:fi_getinfo@@FABRIC_1.3
+   ...
+}
+{
+   FI leak 10
+   Memcheck:Leak
+   match-leak-kinds: definite
+   ...
+   fun:fi_ini
+   fun:fi_getinfo@@FABRIC_1.3
+   ...
+}
```

Comment:
I think these would all be the same if you removed the match-leak-kinds field.

Regardless, we do at some point need to go through and audit/remove/fix all these issues as they all represent a bug, either in our code, or in OFI, or lower down the stack, or a missed close/free call in daos itself.

Reply:
Done, I removed the math-leak-kins field.

My PR is up-to-date with latest master. It is on top of 058fbe5 (06/Oct/20)